src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -1435,6 +1435,7 @@ public override bool OnKeyboardEnter()
                 recoverIndicator = FocusManager.Instance.FocusIndicator;
                 FocusManager.Instance.FocusIndicator = editModeIndicator;
             }
+            UpdateState(true, isPressed);
```

Comment:
Don't we need to consider editMode here??
```suggestion
            UpdateState(editMode, isPressed);
```

Reply:
Actually, no.
Whether it is edit mode or not, Slider instance already gets focused by key and `isFocused` should be always true.
(Even though it is not editMode, the current focused view is Slider because It's the original concept)